### PR TITLE
Fi bo 87 update datatype

### PR DESCRIPTION
**Pull request by Cebox82**

### CODE REVIEW

File: android-app/app/build.gradle [hunk 1]
```diff
@@ -43,4 +43,5 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+    implementation 'de.hdodenhof:circleimageview:3.1.0'
```
Comment (Jonas-Karl) [inline android-app/app/build.gradle:46]: Is this implementation somewhere used?

File: android-app/.idea/assetWizardSettings.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                  </entry>
+                </map>
+              </option>
+            </PersistentState>
+          </value>
+        </entry>
+      </map>
+    </option>
+  </component>
+</project>
```
Comment (Jonas-Karl) [inline android-app/.idea/assetWizardSettings.xml:52]: I don't think this file is used anywhere

File: android-app/app/src/main/java/de/dhbw/ka/se/fibo/models/Category.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+import android.graphics.Color;
+
+import java.util.Random;
+
+
+public enum Category {
+
+
+    RESTAURANT(Color.rgb(216, 49, 91), "Restaurant"), GESUNDHEIT(Color.rgb(89, 178, 34), "Gesundheit"), WOHNEN(Color.rgb(231,74,22), "Wohnen"), HAUSHALT(Color.rgb(33,11,99), "Haushalt"), KULTUR(Color.rgb(53,23,200), "Kultur"), BILDUNG(Color.rgb(234,33,11), "Bildung"), SOZIALLEBEN(Color.rgb(223,22,44), "Sozialleben"), MOBILITÄT(Color.rgb(23,99,22), "Mobilit\u00e4t"), BEKLEIDUNG(Color.rgb(234,200,18), "Bekleidung"), GESCHENK(Color.rgb(66,83,244), "Geschenk"), SONSTIGE(Color.rgb(33,1,9), "Sonstige"), VERSICHERUNG(Color.rgb(23,42,44), "Versicherung");
```
Comment (Jonas-Karl) [inline android-app/app/src/main/java/de/dhbw/ka/se/fibo/models/Category.java:12]: Maybe the enum constants should be english

**Review by Jonas-Karl — COMMENTED** [review]
Added some comments there but overall it looks good to me. just let me know if I somehow missed where the xml file and the new implementation are used

File: android-app/app/src/main/java/de/dhbw/ka/se/fibo/models/Category.java (hunk 3 above)
Comment (jens1o) [reply android-app/app/src/main/java/de/dhbw/ka/se/fibo/models/Category.java:12]: … or rather, they should be using a language variable …

**Review by jens1o — APPROVED** [review]
great, however we need the merge conflicts resolved. Also, the values in the `strings.xml` should be German :)

**Comment by Jonas-Karl** [comment]
Did you check wether we need the files and implementations I asked about?

File: android-app/app/build.gradle (hunk 1 above)
Comment (Cebox82) [reply android-app/app/build.gradle]: Currently not, had only tried something with it

**Pull request merged**